apps/docs/content/components/FAQ/react.mdx

```diff
@@ -433,12 +431,13 @@ render(<App />)
 
 ### FAQGroup
 
-| Name        | Type                                                                                                                           | Default | Description                              |
-| :---------- | :----------------------------------------------------------------------------------------------------------------------------- | :-----: | :--------------------------------------- |
-| `className` | `string`                                                                                                                       |         | Sets a custom class                      |
-| `id`        | `string`                                                                                                                       |         | Sets a custom id                         |
-| `ref`       | `React.RefObject`                                                                                                              |         | Forward a Ref to the underlying DOM node |
-| `children`  | <PropTableValues values={[<a href="#faq-required">FAQ</a>, <a href="#faqgroup-heading">FAQGroup.Heading</a>]} addLineBreaks /> |         | Root element for the FAQGroup component. |
+| Name            | Type                                                                                                                           | Default | Description                                                                   |
+| :-------------- | :----------------------------------------------------------------------------------------------------------------------------- | :-----: | :---------------------------------------------------------------------------- |
+| `className`     | `string`                                                                                                                       |         | Sets a custom class                                                           |
+| `id`            | `string`                                                                                                                       |         | Sets a custom id                                                              |
+| `ref`           | `React.RefObject`                                                                                                              |         | Forward a Ref to the underlying DOM node                                      |
+| `children`      | <PropTableValues values={[<a href="#faq-required">FAQ</a>, <a href="#faqgroup-heading">FAQGroup.Heading</a>]} addLineBreaks /> |         | Root element for the FAQGroup component.                                      |
+| `tabAttributes` | `(children: ReactElement, index: number) => Record<string, unknown>`                                                           |         | Spreads the returned attributes onto the tab that's rendered by the FAQGroup. |
```

Comment:
```suggestion
| `tabAttributes` | `(children: ReactElement, index: number) => Record<string, string>`                                                           |         | Spreads the returned attributes onto the tab that's rendered by the FAQGroup. |
```

Reply:
Could we also add an example of how to use this prop to either Storybook or Documentation please. So that we have a reference point for the user.

Reply:
> as users may want to pass values other than strings

Could you provide a few examples please? I would have thought attributes always need to interpolate back to strings?

Or maybe you're referring to event props? In which case can we be explicit about the exact type here. I.e. `() => void`. I think it would help to be a little clearer in the prop table.

Reply:
Attributes technically do always need to be strings, but as we're working with props in React-land then really the values in this object can be any type.

I don't know of any specific examples, but `unknown` feels like the most accurate type as, under the hood, the user can pass anything they like and it will get forwarded to the `Button`. Even if we make it `Record<string, string>` then the user can still pass whatever they like with a `@ts-ignore` and it will work. I think the fact that it will work makes me feel `unknown` felt accurate. If we were enforcing that the values were strings then `Record<string, string>` would be better in that case.

Using `Partial<ButtonProps>` instead of `Record<string, unknown>` would potentially be more accurate, but I ran into some type issues with the Button component that felt out-of-scope to fix in this PR.

As always, I'll gladly change it if you don't think it's the right choice, I just wanted to give my reasoning 🙂

> Could we also add an example of how to use this prop to either Storybook or Documentation please. So that we have a reference point for the user.

I somehow missed this when writing my above comment, I'll add an example to the docs shortly

Reply:
I see, thanks for clarifying. Let's keep as-is for now...

For additional context on original question though... I see it as two issues:

- We could implement some type narrowing to remove ambiguity around types and prevent need for `unknown`
- because of how manual the types are in our docs we have an opportunity to be more intentional about which types we want users to pass through. E.g. We can advertise that it accepts either a function or string, even if we don't implement narrowing and the implementation detail technically allows for more. 

The thing I'm highlighting is more of an optics issue, where `unknown` is a bit 'ugly' in documentation. I can totally see why we're going with this though, and I think the only option is to use type narrowing if we want to be 100% accurate.